recipes/install-kibana.rb

```diff
@@ -0,0 +1,46 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-kibana
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+elk_info = get_elk_info
+
+kibana_version = elk_info[:kibana_version]
+kibana_checksum = elk_info[:kibana_checksum]
+dl_path = "#{::Chef::Config[:file_cache_path]}/kibana.tar.gz" 
+
+group 'kibana' do
+  append true
+end
+
+user 'kibana' do
+  supports manage_home: true
+  gid 'kibana'
+  shell '/bin/false'
+  home '/opt/kibana'
+end
+
+remote_file "/etc/init.d/kibana" do
```

Comment:
does it make sense to download these files and put them into `files/` directly? I would like to minimize third party dependencies, especially for something as ephemeral-feeling as a gist.